docs/en/reference/tray.md

```diff
@@ -0,0 +1,25 @@
+# Tray
+Optionally, Zettlr can show a tray with the Zettlr icon and a context menu in the system's notification area. When enabled, Zettlr will stay running after closing the editor window.
+
+## Platform support
+- Windows 10
+- MacOS
+- Linux
+
+## Activate Tray
+To activate the tray, open Preferences using either the menu item, the toolbar button or pressing `Cmd/Ctrl+,`. In the Advanced tab, select **Show app in the notification area** (or **Show app in the notification area** when using MacOS).
```

Comment:
This should be:
```
In the Advanced tab, select **Leave app running in the notification area** (or **Show app in the notification
area** when using MacOS).
```

Note the 'Leave app running in the notification area'

Reply:
Thank you. I have update the changes.